### DISCUSSION TITLE
set Template active in custom bin file

### DISCUSSION THREAD

**Discussion by kareemkhalil** [arendst/Tasmota#15828] [Q&A] [answered]
Hello Dears, 
i am starting to compile tasmota and i want to set the template i made to active from compiler i manged to set USER_TEMPLATE with my custom template but i don't know how to set it as active.

**Comment by sfromis** [comment]
By also:
`#define MODULE USER_MODULE`

**Comment by ascillato** [comment, marked as answer]
Hi,

Already answered at https://github.com/arendst/Tasmota/discussions/14847#discussioncomment-2173811
